lab-anthony/greet-cli.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const getGreet = require(__dirname + '/lib/greet');
+
+getGreet.greet();
```

Comment:
This is fine, in the future you may consider doing this instead for more clarity:
const greet = require(__dirname + '/lib/greet').greet;
That way when debugging it is very clear where that function (greet) comes from.